pcsx2/SaveState.cpp

```diff
@@ -518,6 +518,17 @@ class SavestateEntry_EmotionMemory final : public MemorySavestateEntry
 	}
 };
 
+class SavestateEntry_EmotionExtraMemory final : public MemorySavestateEntry
```

Comment:
Yeah this isn't really necessary now. It's was a remnant of when I was trying to avoid bumping the save state version, but it was unavoidable due to needing that extra data for the extra memory state.